website/docs/core-concepts/stacks/templates/datasources.mdx

```diff
@@ -208,95 +244,8 @@ components:
       vars:
         enabled: '{{ (datasource "config-2").config.enabled }}'
 ```
-
-
-<File title="atmos.yaml">
-```yaml
-# https://pkg.go.dev/text/template
-templates:
-  settings:
-    # Enable `Go` templates in Atmos stack manifests
-    enabled: true
-    # https://docs.gomplate.ca
-    # https://docs.gomplate.ca/functions
-    gomplate:
-      # Enable Gomplate functions and datasources in `Go` templates in Atmos stack manifests
-      enabled: true
-      # Timeout in seconds to execute the datasources
-      timeout: 5
-      # https://docs.gomplate.ca/datasources
-      datasources:
-        # 'http' datasource
-        # https://docs.gomplate.ca/datasources/#using-file-datasources
-        ip:
-          url: "https://api.ipify.org?format=json"
-          # https://docs.gomplate.ca/datasources/#sending-http-headers
-          # https://docs.gomplate.ca/usage/#--datasource-header-h
-          headers:
-            accept:
-              - "application/json"
-        # 'file' datasources
-        # https://docs.gomplate.ca/datasources/#using-file-datasources
-        config-1:
-          url: "./config1.json"
-        config-2:
-          url: "file:///config2.json"
-        # `aws+smp` AWS Systems Manager Parameter Store datasource
-        # https://docs.gomplate.ca/datasources/#using-awssmp-datasources
-        secret-1:
-          url: "aws+smp:///path/to/secret"
-        # `aws+sm` AWS Secrets Manager datasource
-        # https://docs.gomplate.ca/datasources/#using-awssm-datasource
-        secret-2:
-          url: "aws+sm:///path/to/secret"
-        # `s3` datasource
-        # https://docs.gomplate.ca/datasources/#using-s3-datasources
-        s3-config:
-          url: "s3://mybucket/config/config.json"
-```
 </File>
 
-
-<dt>`templates.settings.gomplate.datasources`</dt>
-<dd>
-A map of [Gomplate Datasource](https://docs.gomplate.ca/datasources) definitions.
-
-The keys of the map are the datasource names (aliases), which are used in `Go` templates in Atmos stack manifests.
-    For example:
-    <File title="stack.yaml">
-    ```yaml
-    terraform:
-      vars:
-        tags:
-          provisioned_by_ip: '{{ (datasource "ip").ip }}'
-          config1_tag: '{{ (datasource "config-1").tag }}'
-          config2_service_name: '{{ (datasource "config-2").service.name }}'
-    ```
```

Comment:
Why are we deleting this? We should _not_ rely on the upstream documentation, which is a) not searchable from atmos b) not obvious to newcomers. We import more docs, in fact.